roles/bmc_fw_update/tasks/main.yml

```diff
@@ -1,2 +1,85 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2022-2024 Dell Inc, or its subsidiaries.
+
 ---
 # tasks file for bmc_fw_update
+
+- name: Get Firmware Inventory
```

Comment:
This can be a get_bmc_fact role with firmware inventory as an option, so that the info can be fetched individually and the role can be reused here.

Reply:
done